### DISCUSSION TITLE
Insightgraph

### DISCUSSION THREAD

**Discussion by jjohare** [infiniflow/ragflow#2448] [Ideas]
https://arxiv.org/abs/2409.03258
[edit, [now a feature request](https://github.com/infiniflow/ragflow/issues/2455)]

**Comment by KevinHuSh** [comment]
Could you please elaberate on it with a [Feature Request] issue?

**Reply by jjohare** [reply]
done thanks, I don't have implmentation insights on this so I have left that blank but I will keep looking at it and revisit as I learn
